bin/Indexer.py

```diff
@@ -58,7 +108,24 @@
                 continue
             docpath = message.split(" ", -1)[-1]
             paste = PST.get_p_content()
-            print "Indexing :", docpath
+            print "Indexing - "+indexname+" :", docpath
+
+
+            if time.time() - last_refresh > TIME_WAIT: #avoid calculating the index's size at each message
```

Comment:
Should we do it that often? 1 second seems quite intense to me. I don't think it will make much a difference if we do it every second or every let's say 15 min.